v4/weaviate/graphql/neartextbuilder.go

```diff
@@ -1,11 +1,72 @@
 package graphql
 
 import (
-	"encoding/json"
 	"fmt"
 	"strings"
 )
 
+// fldMover is a type representing field names of a move sub query
+type fldMover string
+
+const (
+	fldMoverConcepts fldMover = "concepts"
+	fldMoverForce    fldMover = "force"
+	fldMoverID       fldMover = "id"
+	fldMoverBeacon   fldMover = "beacon"
+	fldMoverObjects  fldMover = "objects"
+)
+
+// MoveParameters to fine tune Explore queries
+type MoveParameters struct {
+	// Concepts that should be used as base for the movement operation
+	Concepts []string
+	// Force to be applied in the movement operation
+	Force float32
+	// Movers is a list of movers used to adjust the serach direction
+	Movers []MoverObject
```

Comment:
I would suggest to change the name of this field to reflect the actual GraphQL field `Movers` -> `Objects` so that it would be aligned with all other our clients and also users would know what's the purpose of this field as they would already relate to `nearText.moveTo.objects`

Reply:
Do you mean changing MoverObject to MoverObjects?